### ISSUE TITLE
markMediaSeen() does not mark as read

### ISSUE THREAD

**Issue by michelmelo** [mgp25/Instagram-API#2630]
## Prerequisites
- You will be asked some questions and requested to provide some information, please read them **carefully** and answer completely.
- Put an `x` into all the boxes [ ] relevant to your issue (like so [x]).
- Use the *Preview* tab to see how your issue will actually look like, before sending it.
- Understand that we will *CLOSE* (without answering) *all* issues related to `challenge_required`, `checkpoint_required`, `feedback_required` or `sentry_block`. They've already been answered in the Wiki and *countless* closed tickets in the past!
- Do not post screenshots of error messages or code.

---

### Before submitting an issue make sure you have:
- [x] [Searched](https://github.com/mgp25/Instagram-API/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [x] [Read the FAQ](https://github.com/mgp25/Instagram-API/wiki/FAQ)
- [x] [Read the wiki](https://github.com/mgp25/Instagram-API/wiki)
- [x] [Reviewed the examples](https://github.com/mgp25/Instagram-API/tree/master/examples)
- [x] [Installed the api using ``composer``](https://github.com/mgp25/Instagram-API#installation)
- [x] [Using latest API release](https://github.com/mgp25/Instagram-API/releases)

### Purpose of your issue?
- [x] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [x] Question
- [ ] Other

---

### For a *bug report*, you must include *code* that will replicate the error, and the *error log/traceback*.

I'm running this code, but then I'm going to see the story, it's still unread. even if the answer is ok status.

Example Code:

```php
try {

	$feed = $ig->story->getReelsTrayFeed();
    $itemsnew = $feed->getTray();
    foreach ($itemsnew as $key => $item) {
        echo "username: ". $item->getUser()->getUsername() ."\n\r"; 
        $userID = $ig->people->getUserIdForName($item->getUser()->getUsername());
        $feed3 = $ig->story->getUserStoryFeed($userID);
        $items = $feed3->getReel()->getItems();
        echo $ig->story->markMediaSeen($items); 
        echo $ig->internal->markStoryMediaSeen($items, null);
        echo count($items) ."--\n\r";
    }
    
} catch (\Exception $e) {
    echo 'Something went wrong: '.$e->getMessage()."\n";
}
```

**Comment by mgp25** [comment]
@michelmelo debug log pls

**Comment by michelmelo** [comment]
`

username: vinhosdeluxo                                                                                                                                                                                                                                                          ?[1;34mGET:  ?[0mhttps://i.instagram.com/api/v1/users/vinhosdeluxo/usernameinfo/                                                                                                                                                                                                ?[1;33m→ 0B?[0m                                                                                                                                                                                                                                                                 ?[0;32m← 200     1.59kB?[0m                                                                                                                                                                                                                                                     ?[0;36mRESPONSE: ?[0m{"user": {"pk": 7010137602, "username": "vinhosdeluxo", "full_name": "Vinhos De Luxo", "is_private": false, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/f0408f32dc3b86633337fede1233447d/5D5AB03F/t51.2885-19/s150x150/26870654_1970466 67546565_896617278223679488_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pic_id": "1707871986028655068_7010137602", "is_verified": false, "has_anonymous_profile_picture": false, "media_count": 101, "geo_media_count": 0, "follower_count": 3413, "following_count ": 941, "following_tag_count": 25, "biography": "Prove e compre os melhores vinhos de Portugal #wine #vinho #vinhosdeportugal #loja #vinhos Portugal \ud83c\uddf5\ud83c\uddf9 Brasil \ud83c\udde7\ud83c\uddf7 WhatsApp: https://wa.me/351964033083", "can_link_entities_in_bio" : true, "biography_with_entities": {"raw_text": "Prove e compre os melhores vinhos de Portugal #wine #vinho #vinhosdeportugal #loja #vinhos Portugal \ud83c\uddf5\ud83c\uddf9 Brasil \ud83c\udde7\ud83c\uddf7 WhatsApp: https://wa.me/351964033083", "entities": [{"hashtag": { "id": 17843999161028266, "name": "loja"}}, {"hashtag": {"id": 17843854744062272, "name": "vinhos"}}, {"hashtag": {"id": 17843785975010171, "name": "vinhosdeportugal"}}, {"hashtag": {"id": 17842453528065849, "name": "wine"}}, {"hashtag": {"id": 17842322281066437, "name":  "vinho"}}]}, "external_url": "http://www.vinhosdeluxo.com/", "external_lynx_url": "https://l.instagram.com/?u=http%3A%2F%2Fwww.vinhosdeluxo.com%2F\u0026e=ATOh-BuqVQ9iqFNHbF80PKvcD5XTz25K0T9LM1C9WB2JS8XcQlH5Px3HTU_R-Ib1o9jx57O55skQ20ra", "has_biography_translation": true,  "can_boost_post": true, "can_see_organic_insights": true, "show_insights_terms": false, "can_convert_to_business": false, "can_create_sponsor_tags": false, "can_be_tagged_as_sponsor": true, "total_igtv_videos": 3, "reel_auto_archive": "on", "usertags_count": 62, "userta g_review_enabled": false, "has_recommend_accounts": false, "has_chaining": true, "hd_profile_pic_versions": [{"width": 320, "height": 320, "url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/d096a6917a9ad801ccf48cc956bb3e29/5D51AA32/t51.2885-19/s320x320/26870654_197046667 546565_896617278223679488_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net"}, {"width": 640, "height": 640, "url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/4b7ecefa53b608716ac667d342be4034/5D6D3C5F/t51.2885-19/s640x640/26870654_197046667546565_896617278223679488_n.jpg?_nc _ht=instagram.flhr7-1.fna.fbcdn.net"}], "hd_profile_pic_url_info": {"url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/36254cb279895d534d6235bf4f5034f3/5D415A8B/t51.2885-19/26870654_197046667546565_896617278223679488_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "width" : 750, "height": 750}, "has_placed_orders": false, "show_shoppable_feed": false, "shoppable_posts_count": 0, "can_be_reported_as_fraud": false, "show_conversion_edit_entry": false, "allowed_commenter_type": "any", "is_video_creator": false, "has_profile_video_feed": fals e, "eligible_shopping_signup_entrypoints": [], "direct_messaging": "UNKNOWN", "fb_page_call_to_action_id": "", "address_street": "", "business_contact_method": "CALL", "category": "Product/Service", "city_id": 110432202311659, "city_name": "Lisbon, Portugal", "contact_ph one_number": "915686554", "is_call_to_action_enabled": false, "latitude": 38.7261, "longitude": -9.1394, "public_email": "michel.melo@vinhosdeluxo.com", "public_phone_country_code": "351", "public_phone_number": "915686554", "zip": "", "can_claim_page": false, "can_cross post_without_fb_token": true, "num_of_admined_pages": 0, "page_id": 141346979867053, "page_name": "Vinhos De Luxo", "is_attribute_sync_enabled": false, "has_business_presence_node": false, "profile_visits_count": 382, "profile_visits_num_days": 7, "fb_page_call_to_action _ix_app_id": 0, "fb_page_call_to_action_ix_label_bundle": {}, "fb_page_call_to_action_ix_partner": "", "fb_page_call_to_action_ix_url": "", "is_call_to_action_enabled_by_surface": null, "instagram_location_id": "", "is_business": true, "account_type": 2, "can_hide_catego ry": false, "can_hide_public_contacts": false, "should_show_category": true, "should_show_public_contacts": true}, "status": "ok"}                                                                                                                                                                                                                                                                                                                                                                                                                              ?[1;34mGET:  ?[0mhttps://i.instagram.com/api/v1/feed/user/7010137602/story/?supported_capabilities_new=%5B%7B%22name%22%3A%22SUPPORTED_SDK_VERSIONS%22%2C%22value%22%3A%2213.0%2C14.0%2C15.0%2C16.0%2C17.0%2C18.0%2C19.0%2C20.0%2C21.0%2C22.0%2C23.0%2C24.0%2C25.0%2C26.0%2C27. 0%2C28.0%2C29.0%2C30.0%2C31.0%2C32.0%2C33.0%2C34.0%2C35.0%2C36.0%2C37.0%2C38.0%2C39.0%2C40.0%2C41.0%2C42.0%2C43.0%2C44.0%2C45.0%2C46.0%2C47.0%2C48.0%2C49.0%2C50.0%2C51.0%2C52.0%2C53.0%2C54.0%2C55.0%2C56.0%2C57.0%2C58.0%22%7D%2C%7B%22name%22%3A%22FACE_TRACKER_VERSION%22%2 C%22value%22%3A%2212%22%7D%2C%7B%22name%22%3A%22segmentation%22%2C%22value%22%3A%22segmentation_enabled%22%7D%2C%7B%22name%22%3A%22COMPRESSION%22%2C%22value%22%3A%22ETC2_COMPRESSION%22%7D%2C%7B%22name%22%3A%22world_tracker%22%2C%22value%22%3A%22world_tracker_enabled%22%7 D%2C%7B%22name%22%3A%22gyroscope%22%2C%22value%22%3A%22gyroscope_enabled%22%7D%5D                                                                                                                                                                                               ?[1;33m→ 0B?[0m                                                                                                                                                                                                                                                                 ?[0;32m← 200     2.8kB?[0m                                                                                                                                                                                                                                                      ?[0;36mRESPONSE: ?[0m{"broadcast": null, "reel": {"id": 7010137602, "latest_reel_media": 1556026312, "expiring_at": 1556112712, "seen": 1556026312, "can_reply": false, "can_reshare": true, "reel_type": "user_reel", "user": {"pk": 7010137602, "username": "vinhosdeluxo", " full_name": "Vinhos De Luxo", "is_private": false, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/f0408f32dc3b86633337fede1233447d/5D5AB03F/t51.2885-19/s150x150/26870654_197046667546565_896617278223679488_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "pr ofile_pic_id": "1707871986028655068_7010137602", "friendship_status": {"following": false, "followed_by": false, "blocking": false, "muting": false, "is_private": false, "incoming_request": false, "outgoing_request": false, "is_bestie": false}, "is_verified": false}, "it ems": [{"taken_at": 1556026302, "pk": 2028418112753065719, "id": "2028418112753065719_7010137602", "device_timestamp": 1556026302313569, "media_type": 1, "code": "BwmY4bKnV73", "client_cache_key": "MjAyODQxODExMjc1MzA2NTcxOQ==.2", "filter_type": 0, "image_versions2": {"c andidates": [{"width": 750, "height": 1334, "url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/a08d87d30d40abcb9790cb254b3faa40/5CC2B003/t51.12442-15/e35/58079967_125532878624435_72747260605467413_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net\u0026ig_cache_key=MjAyODQxODE xMjc1MzA2NTcxOQ%3D%3D.2"}, {"width": 240, "height": 426, "url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/804d2e317957ecacb7d0427054984dc1/5CC28790/t51.12442-15/e35/p240x240/58079967_125532878624435_72747260605467413_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net\u0026ig _cache_key=MjAyODQxODExMjc1MzA2NTcxOQ%3D%3D.2"}]}, "original_width": 750, "original_height": 1334, "caption_position": 0.0, "is_reel_media": true, "timezone_offset": 3600, "xpost_deny_reason": "This story contains another post and can't be shared to Facebook.", "user": { "pk": 7010137602, "username": "vinhosdeluxo", "full_name": "Vinhos De Luxo", "is_private": false, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/f0408f32dc3b86633337fede1233447d/5D5AB03F/t51.2885-19/s150x150/26870654_197046667546565_896617278223679488_n.j pg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pic_id": "1707871986028655068_7010137602", "is_verified": false, "has_anonymous_profile_picture": false, "can_boost_post": true, "can_see_organic_insights": true, "show_insights_terms": false, "reel_auto_archive": "on" , "is_unpublished": false, "allowed_commenter_type": "any"}, "caption_is_edited": false, "boosted_status": "unavailable", "boost_unavailable_reason": "We currently don't support the promotion of stories with tappable elements. Try promoting a different story.", "photo_of _you": false, "caption": null, "fb_user_tags": {"in": []}, "can_viewer_save": true, "organic_tracking_token": "eyJ2ZXJzaW9uIjo1LCJwYXlsb2FkIjp7ImlzX2FuYWx5dGljc190cmFja2VkIjp0cnVlLCJ1dWlkIjoiZGEwOWFjMmRlY2RkNDBiOGEzYTVlYjJlMWU4ZDAyODYyMDI4NDE4MTEyNzUzMDY1NzE5Iiwic2VydmVy X3Rva2VuIjoiMTU1NjEwNzY2MDk2MHwyMDI4NDE4MTEyNzUzMDY1NzE5fDcwMTAxMzc2MDJ8ODFmNTcyNjAyMDEyMjU2NGM4MWI4ZmMwMTFlZjYyNTFiODdjNzYxMDY0MzFlMGNlOTVmZDIxYjNlZjBjNGMwZiJ9LCJzaWduYXR1cmUiOiIifQ==", "expiring_at": 1556112702, "can_reshare": true, "can_reply": false, "story_feed_medi a": [{"x": 0.49068892441842504, "y": 0.48069071846667005, "z": 0, "width": 0.5, "height": 0.5, "rotation": 0.0, "is_pinned": 0, "is_hidden": 0, "media_id": 2027526407933799995, "product_type": "feed"}], "story_is_saved_to_archive": true, "viewers": [{"pk": 5385767040, "u sername": "domainedelacoche", "full_name": "domainedelacoche", "is_private": false, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/ad795b5f230dd06e8a2100cfc885644d/5D706A1F/t51.2885-19/18094961_153042158563034_7305234857867608064_a.jpg?_nc_ht=instagram.fl hr7-1.fna.fbcdn.net", "profile_pic_id": "1502308259254616711_5385767040", "is_verified": false}, {"pk": 1910712054, "username": "will_surf2003", "full_name": "will", "is_private": true, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/104b12eec63f3784be75f1 6deb08e424/5D52ACC5/t51.2885-19/s150x150/47693536_1162768893889799_3965709128789131264_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pic_id": "1949696489769327109_1910712054", "is_verified": false}, {"pk": 8672378468, "username": "matveevphotography", "full_nam e": "Anton Matveev", "is_private": false, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/aaafb65a88a121d6a5f6b8cac2689158/5D58566A/t51.2885-19/s150x150/53192826_2190299154347215_317832905663447040_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pi c_id": "2008387909124987763_8672378468", "is_verified": false}, {"pk": 11793554462, "username": "antonmatveevphotography", "full_name": "\u0421\u0432\u0430\u0434\u0435\u0431\u043d\u044b\u0439 \u0444\u043e\u0442\u043e\u0433\u0440\u0430\u0444 \u0413\u0440\u0443\u0437\u0438 \u044f", "is_private": false, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/6ea987680d59182b67a5244294d540a6/5D6B4322/t51.2885-19/s150x150/53884439_2594222543981787_4890845762952364032_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pic_id": "201 1169109326021088_11793554462", "is_verified": false}, {"pk": 10176032102, "username": "vladimirsmirnov278", "full_name": "\u0412\u043b\u0430\u0434\u0438\u043c\u0438\u0440 \u0421\u043c\u0438\u0440\u043d\u043e\u0432", "is_private": false, "profile_pic_url": "https://instag ram.flhr7-1.fna.fbcdn.net/vp/c33fc166d1e23a12ab4d9cc50f084b7b/5D4284AA/t51.2885-19/s150x150/49465648_1851581014969682_632123631538798592_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pic_id": "1948140331892925610_10176032102", "is_verified": false}], "viewer_co unt": 61, "viewer_cursor": "5", "total_viewer_count": 61, "multi_author_reel_names": [], "supports_reel_reactions": false, "show_one_tap_fb_share_tooltip": true, "has_shared_to_fb": 0}, {"taken_at": 1556026312, "pk": 2028418194877538224, "id": "2028418194877538224_701013 7602", "device_timestamp": 1556026312682002, "media_type": 1, "code": "BwmY5npnV-w", "client_cache_key": "MjAyODQxODE5NDg3NzUzODIyNA==.2", "filter_type": 0, "image_versions2": {"candidates": [{"width": 750, "height": 1334, "url": "https://instagram.flhr7-1.fna.fbcdn.net/ vp/d730fc000d1b39d8b399b983e18c7a86/5CC2C27A/t51.12442-15/e35/56549575_343522699632993_8855865751617488322_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net\u0026ig_cache_key=MjAyODQxODE5NDg3NzUzODIyNA%3D%3D.2"}, {"width": 240, "height": 426, "url": "https://instagram.flhr7-1 .fna.fbcdn.net/vp/b8355661fe9ce9e800a245e95f700bd2/5CC2A3E2/t51.12442-15/e35/p240x240/56549575_343522699632993_8855865751617488322_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net\u0026ig_cache_key=MjAyODQxODE5NDg3NzUzODIyNA%3D%3D.2"}]}, "original_width": 750, "original_heig ht": 1334, "caption_position": 0.0, "is_reel_media": true, "timezone_offset": 3600, "xpost_deny_reason": "This story contains another post and can't be shared to Facebook.", "user": {"pk": 7010137602, "username": "vinhosdeluxo", "full_name": "Vinhos De Luxo", "is_private ": false, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/f0408f32dc3b86633337fede1233447d/5D5AB03F/t51.2885-19/s150x150/26870654_197046667546565_896617278223679488_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pic_id": "1707871986028655068_70101 37602", "is_verified": false, "has_anonymous_profile_picture": false, "can_boost_post": true, "can_see_organic_insights": true, "show_insights_terms": false, "reel_auto_archive": "on", "is_unpublished": false, "allowed_commenter_type": "any"}, "caption_is_edited": false,  "boosted_status": "unavailable", "boost_unavailable_reason": "We currently don't support the promotion of stories with tappable elements. Try promoting a different story.", "photo_of_you": false, "caption": null, "fb_user_tags": {"in": []}, "can_viewer_save": true, "org anic_tracking_token": "eyJ2ZXJzaW9uIjo1LCJwYXlsb2FkIjp7ImlzX2FuYWx5dGljc190cmFja2VkIjp0cnVlLCJ1dWlkIjoiZGEwOWFjMmRlY2RkNDBiOGEzYTVlYjJlMWU4ZDAyODYyMDI4NDE4MTk0ODc3NTM4MjI0Iiwic2VydmVyX3Rva2VuIjoiMTU1NjEwNzY2MDk2MXwyMDI4NDE4MTk0ODc3NTM4MjI0fDcwMTAxMzc2MDJ8MDc5ZmVlMzI5Nzc5 MDFiZDg2MDIyZjdkZWMyMzJkZmQ0MDA4ZGZjNWFmMjgyMjQ1YWIwNTU5NDk2OTQ3NTY3MyJ9LCJzaWduYXR1cmUiOiIifQ==", "expiring_at": 1556112712, "can_reshare": true, "can_reply": false, "story_feed_media": [{"x": 0.501482517262947, "y": 0.46327585551486705, "z": 0, "width": 0.5, "height":  0.5, "rotation": 0.0, "is_pinned": 0, "is_hidden": 0, "media_id": 2028251181517551193, "product_type": "feed"}], "story_is_saved_to_archive": true, "viewers": [{"pk": 2521778280, "username": "puffyzavzay", "full_name": "Dahlia", "is_private": false, "profile_pic_url": "h ttps://instagram.flhr7-1.fna.fbcdn.net/vp/da01d77255328213bc359b7fe2f919f0/5D5E3E6C/t51.2885-19/s150x150/15535220_241021426324621_4896723932503080960_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pic_id": "1432869859536291440_2521778280", "is_verified": false},  {"pk": 5455445552, "username": "nicolasquarcina", "full_name": "Nicola Squarcina", "is_private": false, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/29ceda9628edd4361428e5cd61e9a125/5D570046/t51.2885-19/s150x150/18382061_842849495865239_382394156983726 8992_a.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pic_id": "1513349332001473326_5455445552", "is_verified": false}, {"pk": 5385767040, "username": "domainedelacoche", "full_name": "domainedelacoche", "is_private": false, "profile_pic_url": "https://instagram.f lhr7-1.fna.fbcdn.net/vp/ad795b5f230dd06e8a2100cfc885644d/5D706A1F/t51.2885-19/18094961_153042158563034_7305234857867608064_a.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pic_id": "1502308259254616711_5385767040", "is_verified": false}, {"pk": 1910712054, "userna me": "will_surf2003", "full_name": "will", "is_private": true, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/104b12eec63f3784be75f16deb08e424/5D52ACC5/t51.2885-19/s150x150/47693536_1162768893889799_3965709128789131264_n.jpg?_nc_ht=instagram.flhr7-1.fna.f bcdn.net", "profile_pic_id": "1949696489769327109_1910712054", "is_verified": false}, {"pk": 8672378468, "username": "matveevphotography", "full_name": "Anton Matveev", "is_private": false, "profile_pic_url": "https://instagram.flhr7-1.fna.fbcdn.net/vp/aaafb65a88a121d6a5 f6b8cac2689158/5D58566A/t51.2885-19/s150x150/53192826_2190299154347215_317832905663447040_n.jpg?_nc_ht=instagram.flhr7-1.fna.fbcdn.net", "profile_pic_id": "2008387909124987763_8672378468", "is_verified": false}], "viewer_count": 49, "viewer_cursor": "5", "total_viewer_co unt": 49, "multi_author_reel_names": [], "supports_reel_reactions": false, "show_one_tap_fb_share_tooltip": true, "has_shared_to_fb": 0}], "prefetch_count": 0, "has_besties_media": false, "media_count": 2, "hide_from_feed_unit": true}, "status": "ok"}                                                                                                                                                                                                                                                                                                     ?[1;34mPOST:  ?[0mhttps://i.instagram.com/api/v2/media/seen/?reel=1&live_vod=0                                                                                                                                                                                                  ?[1;33mDATA: ?[0                                                                                                                                                                                                                                                                                      ╚                                                                                                                                                                                                                                                         ?[0;32m← 200     16B?[0m                                                                                                                                                                                                                                                        ?[0;36mRESPONSE: ?[0m{"status": "ok"}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           {"status":"ok"}?[1;34mPOST:  ?[0mhttps://i.instagram.com/api/v2/media/seen/?reel=1&live_vod=0                                                                                                                                                                                   ?[1;33mDATA: ?[0                                                                                                                                                                                                                                                                                      ╚                                                                                                                                                                                                                                                         ?[0;32m← 200     16B?[0m                                                                                                                                                                                                                                                        ?[0;36mRESPONSE: ?[0m{"status": "ok"}
--


`

**Comment by michelmelo** [comment]
@mgp25  
the bug is in 5.0.0 in version v4.1.0already works

**Comment by JRoy** [comment]
Please use dev-master not 5.0 when your report an issue. If you use dev-master does it work?

**Comment by michelmelo** [comment]
not. is only working on v4.1.0

**Comment by mgp25** [comment]
@JRoy Can you verify this?

**Comment by JRoy** [comment]
Turns out this request is not gzipped. Will be fixed in a few minutes.

Regards.

**Comment by giovanniciriello** [comment]
> Turns out this request is not gzipped. Will be fixed in a few minutes.
> 
> Regards.
> 
> Edit: Fixed in [203b10a](https://github.com/mgp25/Instagram-API/commit/203b10a200a027658382e9b81691d02aa14f0630)

I removed `->setIsBodyCompressed(true)` line but still not working: the stories are not marked as seen
